WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -77,6 +77,21 @@
       <%= paginate @posts, param_name: :posts_page, params: { anchor: "posts" } %>
     <% end %>
 
+    <% if !@regulations.empty? %>
+      <h2><%= anchorable "Regulations and Guidelines" %></h2>
+      <% @regulations.each do |reg| %>
+        <div>
+          <%= link_to reg["url"] do %>
+            <h4><%= wca_highlight(reg["id"], query_parts) %></h4>
+          <% end %>
+          <blockquote>
+            <%= wca_highlight(reg["text"], query_parts) %>
```

Comment:
Ah, darn! Yeah, being consistent would be good here, but I'm not sure what the best way to strip html tags would be... perhaps http://stackoverflow.com/a/8632453 would work for us?